### PR TITLE
Add directories to exclusion list && change default repo string

### PR DESCRIPTION
**Pull request by shazbert**
# Description

Updated default string so we can just run and not have to update configuration. Also adds in directories that are not useful to the documentation tool. 

Fixes # 
Node_modules were wigging out the walkfunc, fixed that issue. 

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

I ran it.

## Please also consider improving test coverage whilst working on a certain package

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [n/a] I have commented my code, particularly in hard-to-understand areas
- [hope so] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool 
- [x] My changes generate no new warnings
- [n/a] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Travis with my changes
- [n/a] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: cmd/documentation/documentation.go [hunk 1]
```diff
@@ -32,6 +32,12 @@ const (
 	ContributorFile = "CONTRIBUTORS"
 )
 
+// DefaultExcludedDirectories defines the basic directory exclusion list for GCT
+var DefaultExcludedDirectories = []string{".github",
+	".git",
+	"node_modules",
+	".vscode"}
```
Comment (xtda) [inline cmd/documentation/documentation.go]: Can you also add .idea to this list

Comment (shazbert) [reply cmd/documentation/documentation.go]: 👍

File: cmd/documentation/documentation.go [hunk 2]
```diff
@@ -77,15 +108,33 @@ type Attributes struct {
 }
 
 func main() {
-	verbose := flag.Bool("v", false, "Verbose output")
-
+	flag.BoolVar(&verbose, "v", false, "Verbose output")
+	flag.StringVar(&toolDir, "tool", "", "pass in the documentation tool directory if outside tool folder")
```
Comment (thrasher-) [inline cmd/documentation/documentation.go]: Please change "tool" to "tooldir", apart from that have tested it and works well!

File: cmd/documentation/documentation.go [hunk 3]
```diff
@@ -77,15 +108,33 @@ type Attributes struct {
 }
 
 func main() {
-	verbose := flag.Bool("v", false, "Verbose output")
-
+	flag.BoolVar(&verbose, "v", false, "Verbose output")
```
Comment (thrasher-) [inline cmd/documentation/documentation.go]: Please also keep consistency with the usage flags (Verbose output vs lowercase pass)

**Pull request merged**